README.md

```diff
@@ -1 +1,55 @@
-TODO: add read me
+# Dashboard Angular Application
+
+The following application is an Angular application with the DevExpress Web Dashboard Angular Component inside. 
+
+The Web Dashboard can be rendered on the client-side, while having data fed to it by an ASP.NET Core / ASP.NET MVC back-end. This means you can:
+
+- Publish Web Dashboard as a part of applications based on Angular.
+- Integrate a dashboard into any place that supports HTTP requests (e.g., see the [GitHub Pages Demo](https://devexpress.github.io/web-dashboard-demo/)).
```

Comment:
It is really any place. The only requirement is to support HTTP requests. It can be sites, applications and so on. I cannot be bore specific here, I do not know where a customer wants to embed Web Dashboard.

Reply:
Would it be too specific in this case if we were to say "websites and web applications"? For example:

```suggestion
- Integrate a dashboard into websites and web applications that support HTTP requests (e.g., see the [GitHub Pages Demo](https://devexpress.github.io/web-dashboard-demo/)).
```